frontend/src/views/Raid.vue

```diff
@@ -75,14 +74,14 @@
         <div class="row">
           <div class="col-xs-12 col-sm-12 col-lg-6 weap-box">
             <span class="raid-title-section bold">
-              <span>Weapon
+              <span>{{$t('raid.weapon')}}
                 <Hint
                 text="Your weapon multiplies your power<br>
```

Comment:
Missing translation for tooltip here

Reply:
resolved

Reply:
Oh I actually meant the ` text="Your weapon multiplies your power<br>`

Reply:
I thought so! :D 
I did that here: https://github.com/CryptoBlades/cryptoblades/pull/842/commits/acded724aefefabbac4aef3b73dde3c8f3c345ad